### ISSUE TITLE
Commented out "Related topics" section

### ISSUE THREAD

**Issue by NextTurn** [dotnet/docs#11283]
https://github.com/dotnet/docs/blob/4781d55c5e53c0bae8e36766c1d879672d699456/docs/core/tutorials/vb-with-visual-studio.md#L79-L84

It is worst practice to comment out something without adding a TODO tag.

**Comment by mairaw** [comment]
@rpetrusha can you look at this feedback?

**Comment by rpetrusha** [comment]
Thanks for noticing and pointing this out, @NextTurn. Your observation about the importance of a TODO tag is a good one -- I reviewed the history to find out when the text was commented out, only to discover that it was done by me in the [first commit of the original PR](https://github.com/dotnet/docs/pull/2811/commits/e9ebbab1e01cadbc19490cb15772029d50791125) that created the file. Evidently, the TODO comment should have read something like "Modify and enable these links when the VB topics are available." That might have prevented this work item from falling through the cracks. Fortunately, you noticed my error of omission, and I've addressed it in #11293.
